adapters/rack/example/trace_demonstration2.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+# Copyright 2020 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+require 'rubygems'
+require 'bundler/setup'
+
+Bundler.require
+
+# setup fake rack application:
+builder = Rack::Builder.new
+app = ->(_env) { [200, { 'Content-Type' => 'text/plain' }, ['All responses are OK']] }
+builder.run app
+
+# demonstrate integration using config[:application] instead of 'builder.use':
```

Comment:
I'm not sure that this demonstration is applicable any longer. We require an explicit `use` like the original Datadog instrumentation does.

Reply:
I have updated the example to integrate explicitly (`use`).

Note that the `config[:application]` behavior should be similar to the `MiddlewareNamePatcher` https://github.com/DataDog/dd-trace-rb/blob/9301de22e173dcbd6a4870b6ed1195e71018086c/lib/ddtrace/contrib/rack/patcher.rb#L90